Sources/ArcGISToolkit/Components/Offline/PreplannedListItemView.swift

```diff
@@ -75,7 +84,8 @@ struct PreplannedListItemView: View {
     }
     
     @ViewBuilder private var deleteButton: some View {
-        if model.status.allowsRemoval {
+        if model.status.allowsRemoval,
+           !selectedMapFromThisModel {
```

Comment:
One problem with not allowing deleting when an area is selected is that, if someone wants to delete all the area from the local disk, he has to set `selectedMap = nil` at app level, or otherwise there would always be a single area that cannot be deleted.

People have agreed on this "unselect" can be handled at app level.